static/css/restyle/restyle.less

```diff
@@ -2436,3 +2436,20 @@ body.windows .install-shell .platform.mac {
 #ui-datepicker-div {
   display: none;
 }
+
+// Override weird impala styles that make hovers for incompatible add-ons
+// jump around (https://github.com/mozilla/addons-server/issues/3203)
+.addon.hovercard,
+.items .item.incompatible {
+  .extra .button.not-available {
+    display: inline-block;
+    opacity: hidden;
```

Comment:
Oops, yes. I was playing with opacity then went back to visibility but
messed it up. Thanks!

- tofumatt

On 5 August 2016 at 19:12:57, Mathieu Pillard (notifications@github.com)
wrote:

> In static/css/restyle/restyle.less
> <https://github.com/mozilla/addons-server/pull/3226#discussion_r73736295>:
>
> > @@ -2436,3 +2436,20 @@ body.windows .install-shell .platform.mac {
> >  #ui-datepicker-div {
> >    display: none;
> >  }
> > +
> > +// Override weird impala styles that make hovers for incompatible add-ons
> > +// jump around (https://github.com/mozilla/addons-server/issues/3203)
> > +.addon.hovercard,
> > +.items .item.incompatible {
> > +  .extra .button.not-available {
> > +    display: inline-block;
> > +    opacity: hidden;
>
> Did you mean visibility: hidden ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/addons-server/pull/3226/files/99f23dd35c75a0dcec20b1115b49e24dc5e95559#r73736295>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFi974cDVf_bgOsa5QpbOl9wiG-OAYaks5qc30pgaJpZM4Jd5Tw>
> .
>